runtime/gc_glue_java/ObjectModelDelegate.hpp

```diff
@@ -315,6 +315,26 @@ class GC_ObjectModelDelegate
 		return J9GC_CLASS_IS_ARRAY(clazz);
 	}
 
+	/**
+	 * Get the preserved hashcode offset of forwarded header.
+	 *
+	 * @param[in]   Pointer to forwardedHeader the MM_ForwardedHeader instance encapsulating the object
+	 */
+	MMINLINE uintptr_t
+	getPreservedHashcodeOffset(MM_ForwardedHeader *forwardedHeader)
+	{
+		J9Class *clazz = _arrayObjectModel->getPreservedClass(forwardedHeader);
+		uintptr_t hashOffset;
+
+		if (J9GC_CLASS_IS_ARRAY(clazz)) {
```

Comment:
This is ugly but couldn’t find a better way to ask if an object is Indexable or not. `isIndexable` is not reachable from `_arrayObjectModel` nor `_mixedObjectModel`.